src/WallsComputation.cpp

```diff
@@ -21,30 +21,30 @@ WallsComputation::WallsComputation(const Settings& settings, const LayerIndex la
  * This function is executed in a parallel region based on layer_nr.
  * When modifying make sure any changes does not introduce data races.
  *
- * generateInsets only reads and writes data for the current layer
+ * generateWalls only reads and writes data for the current layer
  */
-void WallsComputation::generateInsets(SliceLayerPart* part)
+void WallsComputation::generateWalls(SliceLayerPart* part)
 {
-    size_t inset_count = settings.get<size_t>("wall_line_count");
+    size_t wall_count = settings.get<size_t>("wall_line_count");
     const bool spiralize = settings.get<bool>("magic_spiralize");
     if (spiralize && layer_nr < LayerIndex(settings.get<size_t>("initial_bottom_layers")) && ((layer_nr % 2) + 2) % 2 == 1) //Add extra insets every 2 layers when spiralizing. This makes bottoms of cups watertight.
     {
-        inset_count += 5;
+        wall_count += 5;
     }
     if (settings.get<bool>("alternate_extra_perimeter"))
     {
-        inset_count += ((layer_nr % 2) + 2) % 2;
+        wall_count += ((layer_nr % 2) + 2) % 2;
     }
 
-    if (inset_count == 0)
+    if (wall_count == 0)
     {
-        part->insets.push_back(part->outline);
+        part->spiral_insets.push_back(part->outline);
```

Comment:
I'd say that the spiralize path should not get 1/2 line width offset if the Wall Line Count is set to 0, right? So this also needs to be inset by half a line width again here.